ci/push_images

```diff
@@ -4,7 +4,7 @@
 set -euo pipefail
 IFS=$'\n\t'
 
-pgversions='10 11 12 13 14 15 16'
+pgversions='10 11 12 13 14 15 16 17'
```

Comment:
Please delete 10 and 11 here since their support has been dropped

Reply:
Fixed